extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -335,11 +337,23 @@ export async function getDefaultPublishDeploymentOptions(project: ISqlProject):
 
 	// this option needs to be true for same database references validation to work
 	if (project.databaseReferences.length > 0) {
-		deploymentOptions.includeCompositeObjects.value = true;
+		deploymentOptions.optionsMapTable[constants.IncludeCompositeObjects].value = true;
 	}
 	return result.defaultDeploymentOptions;
 }
 
+/*
+* Converts first charater of each key to upper case
+*/
+export function convertKeysToUpperCase<T>(options: { [key: string]: T }): { [key: string]: T } {
+	let newObj: { [key: string]: T } = {};
+	Object.entries(options).forEach(function (option) {
+		const key = option[0].charAt(0).toUpperCase() + option[0].slice(1);
+		newObj[key] = option[1];
```

Comment:
what is option[1]? Is this the checked value?

Please update the comment for the function to be more clear.